.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Bug report
+about: Share about things that are not working as expected
+
+---
+
+/kind bug
+
+**Bug description:**
+[Describe the bug in detail including how to reproduce it. Please include any relevant output or screenshots.]
+
+**System information:**
+[Please include your OS, Kubernetes, and kubectl versions as well as anything else that is relevant for us to reproduce or understand the issue.]
```

Comment:
could you break this sentence into bullets for each item? It would be great to include:

- Python version (`python --version `):
- Kubernetes Python client version (`pip freeze | grep kubernetes`): 
- Kubernetes cluster version (`kubectl version`): 
- OS:
- Anything else that is relevant for us to reproduce or understand the issue (e.g. other packages being used `pip freeze`):

Reply:
Fixed.